lang/lang-import.sh

```diff
@@ -171,6 +188,55 @@ if [ "$LNG" = "nl" ]; then
  sed -i 's/\xc3\x85/A/g' $LNG'_filtered.po'
 fi
 
+if [ "$LGN" = "sv" ]; then
+#repace '�' with 'Aa'
+sed -i 's/\xc3\x85/Aa/g' $LNG'_filtered.po'
```

Comment:
since we are translating one input character into 2 output ones, can't we run into some formatting issues?

Reply:
We have these for `öäuß` in German as well. This part in `lang-import.sh` is a safety net to avoid characters that cannot be shown.
The `lang-check.py` will find too long messages and if we like, we can even prevent from building the firmware to avoid LCD glitches caused by too long messages.